presto-main/src/main/java/io/prestosql/operator/aggregation/AbstractMinMaxAggregationFunction.java

```diff
@@ -26,9 +26,12 @@
 import io.prestosql.operator.aggregation.AggregationMetadata.AccumulatorStateDescriptor;
 import io.prestosql.operator.aggregation.state.BlockPositionState;
 import io.prestosql.operator.aggregation.state.BlockPositionStateSerializer;
-import io.prestosql.operator.aggregation.state.NullableBooleanState;
-import io.prestosql.operator.aggregation.state.NullableDoubleState;
-import io.prestosql.operator.aggregation.state.NullableLongState;
```

Comment:
Should we get rid of these classes? There shouldn't be any reason to keep them around now that there's a replacement, no?

Reply:
They are used in other places where the new classes won't work